modules/fundamental/src/sbom/service/sbom.rs

```diff
@@ -494,6 +496,18 @@ async fn package_from_row(
 
     for purl in row.purls {
         if let Ok(dto) = serde_json::from_value::<PurlDto>(purl) {
+            let cp = CanonicalPurl {
```

Comment:
oh, yeah, I forgot we _also_ have `PurlDto`.  Yo dawg, I heard you liked pURLs, so I put a pURL in your pURL so you can pURL while you pURL.

_sigh_

Reply:
one purl to rule them all, one purl to bind them ....